src/main/java/com/flightstats/hub/metrics/MetricsConfig.java

```diff
@@ -0,0 +1,145 @@
+package com.flightstats.hub.metrics;
+
+import com.flightstats.hub.app.HubVersion;
+import lombok.Getter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Properties;
+
+import static com.flightstats.hub.app.HubHost.getLocalName;
+import com.flightstats.hub.app.HubProvider;
+
+public class MetricsConfig {
+    private final Logger logger = LoggerFactory.getLogger(MetricsConfig.class);
+
+    private Properties properties;
+
+    @Getter
+    private final String role = "hub";
+
+    @Getter
+    private final String team = "ddt";
+
+    @Getter(lazy = true)
+    private final String env = safeGetProperty("app.environment", "dev");
```

Comment:
I would probably use a builder pattern for this instead of deferring until invocation with lazy.
Cuz:
A. failing early is better
B. logic in a a getter/setter is considered an anti-pattern by some (I'm like 70/30 on this)